groups/steering/OWNERS

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+options:
+    no_parent_owners: true
+
+approvers:
+    committee-steering
```

Comment:
No approvers until we can ensure subdirs of groups can't create arbitrary groups
```suggestion
```

Reply:
Fixed.